server/services/SsotToRobotParsing/SsotToRobotParser.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @category Client
+ * @module
+ */
+
+/* eslint-disable dot-notation */
+
+/**
+ * @description Checks whether the given element is of type instruction and contains rpa attributes
+ * @param {Object} currentElement Element to check
+ * @returns {bool} Value specifies if object is of type instruction and contains rpa attributes
+ */
+const isAnRpaInstruction = (currentElement) =>
```

Comment:
I'd prefer if you would use curly braces for the content of that arrow function

Reply:
That wouldn't be EsLint compliant. This is how the syntax should be.

Reply:
Why that? A couple of lines later you are using in fact arrow functions with the curly braces

Reply:
Arrow functions without curly braces include an implicit return. Below we use explicit returns because we do more in the function than just returning somethin. Thats the difference